parse_date.re

```diff
@@ -771,6 +771,42 @@ static timelib_long timelib_parse_tz_cor(char **ptr)
 	return 0;
 }
 
+static timelib_long timelib_parse_tz_minutes(char **ptr, timelib_time *t)
+{
+	timelib_long retval = TIMELIB_UNSET;
+	char *begin = *ptr;
+
+	// First character must be +/-.
```

Comment:
In the past, using ``//`` for comments used to cause issues with certain compilers, that's why I used ``/*`` .. ``*/`` everywhere. PHP runs on a whole bunch of ancient systems with ancient compilers, but I am not sure if it's still worth care about these.

Reply:
Changed to be consistent.